### PR TITLE
tapering_window.py is updated

### PR DESCRIPTION
**Pull request by jithin-ak**
please check the update file.

### CODE REVIEW

File: tapering_window.py [hunk 1]
```diff
@@ -72,12 +70,3 @@ def tapering_window(tseries,method="Hanning"):
         tseries_tapered=tseries*wind_kaiser
         print("kaiser window is applied.....")
     return tseries_tapered
-
-Data_Path="/home/jithin/Desktop/ADCP_Processing_Tool/adps-main/tests/BGS11000.000"
```
Comment (subeeshmp) [inline tapering_window.py]: Hey, 

It is throwing a error below..

NameError: name 'press' is not defined

I would suggest commenting the below part from this code. You can  write as a separate code to test the tapering function.

Comment (jithin-ak) [reply tapering_window.py]: I have removed the testing part from this code. I think the pull request is not yet merged. @shehulthinvrex please check.

Comment (p-amol) [reply tapering_window.py]: Ok, the revised code is working for me.

Comment (p-amol) [reply tapering_window.py]: Subeesh, if the code is working in your pc. You can resolve the conversation.

Comment (subeeshmp) [reply tapering_window.py]: It is not yet merged i think. I am getting same error after pulling. 
"tapered_series=tapering_window(press)" This part is still there in the code in main folder.

**Review by shehulthinvrex — APPROVED** [review]

**Comment by p-amol** [comment]
Subeesh and Jenson, 

If the issue is resolved and there are no other comments, we can merge the pull request.

**Comment by shehulthinvrex** [comment]
Ok

**Pull request merged**